includes/class-wc-payments-token-service.php

```diff
@@ -176,4 +185,20 @@ public function woocommerce_payment_token_set_default( $token_id, $token ) {
 			}
 		}
 	}
+
+	/**
+	 * Controls the output for SEPA on the my account page.
+	 *
+	 * @param  array            $item         Individual list item from woocommerce_saved_payment_methods_list.
```

Comment:
Doc-block alignment.

Reply:
I meant to add one more space in `$item`, so we get to the following formatting:
```php
	/**
	 * Controls the output for SEPA on the my account page.
	 *
	 * @param  array                          $item                   Individual list item from woocommerce_saved_payment_methods_list.
	 * @param  WC_Payment_Token $payment_token The payment token associated with this method entry.
	 * @return   array                                                      Filtered item
	 */
 * ```

Reply:
Woops. Fixed. Added spaces so variable and description are aligned vertically.